lib/mayaUsd/resources/scripts/mayaUsdOptions.py

```diff
@@ -41,10 +56,28 @@ def convertOptionsDictToText(optionsDict):
     """
     Converts options to text with column-separated key/value pairs.
     """
-    optionsList = ['%s=%s' % (name, value) for name, value in optionsDict.items()]
+    optionsList = ['%s=%s' % (name, _convertValueToText(value)) for name, value in optionsDict.items()]
     return ';'.join(optionsList)
 
 
+def _convertValueToText(value):
+    # Unfortunately, for historical reasons, text lists have used
+    # comma-separated values, while floating-point used space-separated values.
+    if isinstance(value,list):
```

Comment:
We need to support backward compatibility with existing user prefs or scripts, so unfortunately we need to support both space and comma as separator, for different data types.